documentation/dsi-contingency-fallback.md

```diff
@@ -41,9 +41,9 @@ This does not end any sessions cached by DSI (assuming DSI is live).
 
 ### Configuration
 
-Adjust the length of time before an EmergencyLoginKey expires with `EMERGENCY_LOGIN_KEY_DURATION` in [`HiringStaff::SignIn::Email::SessionsController`](app/controllers/hiring_staff/sign_in/email/sessions_controller.rb)
+Adjust the length of time before an EmergencyLoginKey expires with `EMERGENCY_LOGIN_KEY_DURATION` in [`HiringStaff::SignIn::Email::SessionsController`](app/controllers/publishers/sign_in/email/sessions_controller.rb)
```

Comment:
I still see `HiringStaff` here

Reply:
Just saw it a minute ago as well, just fixed 😅